spec/requests/endpoints/token_spec.rb

```diff
@@ -15,8 +15,7 @@
   it "respond with correct headers" do
     post token_endpoint_url(code: @authorization.token, client: @client)
 
-    expect(headers["Pragma"]).to eq("no-cache")
-    expect(headers["Cache-Control"]).to be_in(["no-store", "private, no-store"])
+    expect(headers["Cache-Control"]).to be_in(["no-store", "no-cache, no-store", "private, no-store"])
```

Comment:
Bit late here but, given that `Pragma: no-cache` is always present here, would the equivalent not be:

```ruby
    expect(headers["Cache-Control"]).to be_in(["no-store, no-cache", "private, no-store, no-cache"])
```